x-pack/test/security_solution_cypress/cypress/support/es_archiver.ts

```diff
@@ -54,10 +54,34 @@ export const esArchiver = (
     baseDir: '../es_archives',
   });
 
+  const ftrEsArchiverInstance = new EsArchiver({
+    log,
+    client,
+    kbnClient,
+    baseDir: '../../functional/es_archives/security_solution',
+  });
+
   on('task', {
-    esArchiverLoad: async ({ archiveName, ...options }) =>
-      esArchiverInstance.load(archiveName, options),
-    esArchiverUnload: async (archiveName) => esArchiverInstance.unload(archiveName),
+    esArchiverLoad: async ({ archiveName, type = 'cypress', ...options }) => {
+      if (type === 'cypress') {
+        esArchiverInstance.load(archiveName, options);
+      } else if (type === 'ftr') {
+        ftrEsArchiverInstance.load(archiveName, options);
+      } else {
+        throw new Error('It is not possible to load the archive.');
+      }
+      return null;
```

Comment:
Nit: these `return`s probably aren't necessary

Reply:
Ah, good to know! We're no longer returning the results of `load/unload`, though; we should probably bring that back, yes?

Reply:
Tbh I believe is not needed in the way we are doing it.

Reply:
I suppose it's not needed, but we are now changing behavior: previously the task would use the returned promise from `load/unload` to determine success/failure, but now that we always return `null` it will always be marked "successful" (except in your `else` condition, of course).

Reply:
Yeah you are right, returning null fails but in an extremely ugly way. Fixed pushed.